src/renderers/shared/fiber/ReactFiberCommitWork.js

```diff
@@ -300,17 +300,19 @@ module.exports = function<T, P, I, TI, C>(
     switch (finishedWork.tag) {
       case ClassComponent: {
         const instance = finishedWork.stateNode;
-        let error = null;
+        let errors = [];
```

Comment:
The reason this isn't an array is because we only need to capture the first error that's thrown, not all of them. (Or the last error, not sure it's super important. Opinions, @gaearon?)